services/traction/api/db/models/base.py

```diff
@@ -23,6 +27,14 @@ class Config(pydantic.BaseModel.Config):
 class BaseModel(SQLModel, BaseSchema):
     __mapper_args__ = {"eager_defaults": True}
 
+    @classmethod
+    def tenant_select(cls, fallback_tenant_id: UUID = None) -> Select:
```

Comment:
Do not like this... I know you are protecting from developer error, but we should be catching that in the review - that the model class is not using the TenantScopedModel. If a developer has an IDE that uses intellisense or the like, this will appear as a valid method to call, and it shouldn't be.

Reply:
I just want someone who copy pastes an example to not get a 'method not found' and instead the code tells them what their issue is. 

@loneil @popkinj, thoughts?